bigquery/tests/unit/test_enums.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import enum
+import mock
+import pytest
+import six
+
+
+@pytest.fixture
+def enum_under_test():
```

Comment:
I imagine we'll want some other enums in the future. How about we make a `bigquery/tests/unit/enums` directory like we did for models?

Reply:
And why not... :+1: